README.rst

```diff
@@ -83,6 +82,62 @@ After running `aws-mfa`, your credentials file would read:
     aws_secret_access_key = <POPULATED_BY_AWS-MFA>
     aws_security_token = <POPULATED_BY_AWS-MFA>
 
+The default naming convention for the credential section can be overriden by using the ``--long-term-suffix`` and
+``--short-term-suffix`` command line arguments. For example, in a multi account scenario you can have one AWS account
```

Comment:
Awesome thanks. Can you give me a play by play of what your workflow would look like? Are you using `aws-cli`, `boto3`, etc?

I'll test this tonight and then approve. Thanks for your contribution!